apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -419,6 +419,11 @@ public override void Prepare()
         public Task<long> ExecutePartitionedUpdateAsync(CancellationToken cancellationToken = default) =>
             CreateExecutableCommand().ExecutePartitionedUpdateAsync(cancellationToken);
 
+        /// <summary>
+        /// Executes this command to get instance endpointUri.
+        /// </summary>
+        public Task<string> ExecuteGetInstanceEndpointUriAsync() => CreateExecutableCommand().GetInstanceEndpointUriAsync();
```

Comment:
I don't think this belongs in the `SpannerCommand`, least of all public.

Reply:
Moved it to `InstanceHostManager`.